SwanProjects/bin/swan_env

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+#DEFINITIONS:
+# STACK: LCG/CMSSW
+# RELEASE: version of the stack ex (LCG_99)
+# PLATFORM:  usually os and compiler version
+# USER_SCRIPT: customized bash script provided by the user
+# CWD: working directory path
+PARAMETERS=($@) #WARNING: empty options disappear
+
+#Checking if jq is installed
+if ! [ -x "$(command -v jq)" ]; then
+  echo 'Error: jq is not installed.' >&2
+  sleep 60
+  exit 1
+fi
+
+#Checking if we have more that 2 parameters
+if [[ $# -lt 2 ]] ; then
+  echo 'Error: project name and commands required.' >&2
```

Comment:
What is `commands` here?

Reply:
explained better in the documentation of the script, 
is the command(s) to be executed inside the project environment.